src/components/filteredPage.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { BiLeftArrowCircle } from 'react-icons/bi';
+import { date } from '../redux/slices/covidSlice';
+import style from './filteredPage.module.css';
+
+const FilteredPage = () => {
+  const state = useSelector((state) => state.covid);
+  let countries;
+  if (state.data.dates) {
+    countries = state.data.dates[date].countries;
+  }
```

Comment:
- [ ] The filter should be a stateless component. Kindly use a `state lifting` technique in the filter component as per the project requirements.
![Screenshot from 2021-09-07 04-42-39](https://user-images.githubusercontent.com/20245164/132271613-11e2a545-73f8-4f4e-a792-b5ab3ff37791.png)

Reply:
Could you please explain this a bit better this requirement, please? I don't quite understand the objective difference between using state lifting and just filtering data directly. What is the difference between them to the project? Am I just supposed to pass info down to the filtered page as props?